src/collection/components/CollectionOrBundleEdit.tsx

```diff
@@ -324,6 +323,11 @@ const CollectionOrBundleEdit: FunctionComponent<
 		currentCollection: null,
 		initialCollection: null,
 	});
+	const isContributor = !!(collectionState.currentCollection?.contributors || []).find(
+		(contributor) => contributor.profile_id === user?.profile?.id
+	);
+	const isSharedWithOthers =
+		!isContributor && !!(collectionState.currentCollection?.contributors?.length || 0 > 0);
```

Comment:
No need to cast a boolean to a boolean
```suggestion
	const isSharedWithOthers =
		!isContributor && (collectionState.currentCollection?.contributors?.length || 0 > 0);
```

Reply:
Typescript complains about it not being of being type boolean 
![image](https://github.com/viaacode/avo2-client/assets/113341869/6e003e36-d33e-4fa8-a084-2d815c083459)